MetaMask/metamask-extension
# UX: Updated nfts detection notice to use Banner

This PR is to update the NftsDetectionNotice Component to use the `Banner` component

* Fixes #20052 

## Screenshots


### Before
![Screenshot 2023-07-17 at 4 34 37 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/a6d3470d-fc45-487b-b337-cd1d6c7ac1d9)


### After
![Screenshot 2023-07-17 at 4 38 08 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/15d4e1aa-5b16-436c-a6ea-c774ba7f03cc)


## Manual Testing Steps

- Switch to Mainnet
- Keep the NFT detection turned off
- Check the Banner in NFTs Tab

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
